packages/bottom-tabs/src/types.tsx

```diff
@@ -288,6 +290,11 @@ export type BottomTabNavigationConfig = {
    * Style object for the component wrapping the screen content.
    */
   sceneContainerStyle?: StyleProp<ViewStyle>;
+
+  /**
+   * Bottom tab-bar position on the window.
+   */
+  tabBarPosition?: TabBarPosition;
```

Comment:
This needs to be an `option` and not a prop. Let's it to `BottomTabNavigationOptions`

Reply:
Alright